machines/sim_filelist.mk

```diff
@@ -2,6 +2,7 @@
 # simulation. These augment the sythesizable files in core.include.
 
 VHEADERS += $(BSG_MANYCORE_DIR)/testbenches/common/v/bsg_manycore_mem_cfg_pkg.v
+VHEADERS += $(BSG_MANYCORE_DIR)/testbenches/common/v/bsg_manycore_profile_pkg.v
```

Comment:
You'll need to file a separate PR for these in F1 or include them in arch_filelist.mk

Reply:
Yes thanks for letting me know, will do once we plan to move forward with this PR.